brayns/io/MorphologyLoader.cpp

```diff
@@ -67,6 +188,7 @@ bool MorphologyLoader::_importMorphology(
     const size_t simulationOffset,
     float& maxDistanceToSoma)
 {
+    const bool useMetaballs = _geometryParameters.getMetaballsGridSize() != 0;
```

Comment:
+1 for readability: make this a  bool _useMetaballs() const func. and use it everywhere instead of _geometryParameters.getMetaballsGridSize() != 0